hack/verify-codegen.sh

```diff
@@ -40,7 +40,7 @@ cp -a "${REPO_ROOT_DIR}/config"/* "${TMP_DIFFROOT}/config"
 
 # TODO(mattmoor): We should be able to rm -rf pkg/client/ and vendor/
 
-"${REPO_ROOT_DIR}/hack/update-codegen.sh"
+"${REPO_ROOT_DIR}/hack/update-codegen.sh" --generate-protobufs
```

Comment:
does our diff below cover the pertinent directories?

Reply:
Added the test directory there. pkg is covered already.